### PR TITLE
Add sha/asm/keccak1600-avx512.pl.

### PR DESCRIPTION
**Pull request by dot-asm**
[skip ci]

### CODE REVIEW

**Comment by dot-asm** [comment]
> I like the comments

This one took a *lot* of concentration [and some others are in work, which is why I was unresponsive lately]...

**Comment by dot-asm** [comment]
Pushed minor typo fixes in commentaries. Since no code is modified I consider approval as still valid *unless* objection is risen.

File: crypto/sha/asm/keccak1600-avx512.pl [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+.align	32
+__KeccakF1600:
+	lea		iotas(%rip),%r10
+	mov		\$24,%eax
+	jmp		.Loop_avx512
+
+.align	32
+.Loop_avx512:
+	######################################### Theta
+	#vpermq		$A00,@Theta[0],$A00
```
Comment (bernd-edlinger) [inline crypto/sha/asm/keccak1600-avx512.pl]: can you explain why this is commented out?

Comment (dot-asm) [inline crypto/sha/asm/keccak1600-avx512.pl]: Humans tend to appreciate symmetry. Originally code was written as "we start by re-arranging all elements of input vectors". But truth is that $A00 is order that doesn't need rearrangement. Even though it was apparent all along it was added to satisfy the human urge for symmetry. And now it's kept as commentary, to satisfy the drift :-)

**Comment by dot-asm** [comment]
> SImple comment fix shouldn't impact the approval

In situation like this there is visible commit, and it's only appropriate to clarify what is it. And I did write that it would take explicit *objection* to repeal the approval. So what's the point of remark? The only possible point is implied suggestion that comment fix *can't* impact approval. This I would have to disagree with. Not only it *can* void approval (there is no way of knowing in advance), contributor is even in position to *demand* re-approval. This is basically the whole idea behind "it would take objection [from any of reviewers] to repeal the approval", more specifically to distribute power of taking the decision evenly.

**Pull request closed without merging**